kratos/solving_strategies/convergencecriterias/convergence_criteria.h

```diff
@@ -77,25 +59,30 @@ class ConvergenceCriteria
     ///@name Type Definitions
     ///@{
 
+    /// Pointer definition of ConvergenceCriteria
+    KRATOS_CLASS_POINTER_DEFINITION(ConvergenceCriteria);
+
+    /// Data type definition
     typedef typename TSparseSpace::DataType TDataType;
+    /// Matrix type definition
     typedef typename TSparseSpace::MatrixType TSystemMatrixType;
+    /// Vector type definition
     typedef typename TSparseSpace::VectorType TSystemVectorType;
-
+    /// Local system matrix type definition
     typedef typename TDenseSpace::MatrixType LocalSystemMatrixType;
+    /// Local system vector type definition
     typedef typename TDenseSpace::VectorType LocalSystemVectorType;
 
-    //typedef Dof<double> TDofType;
+    /// DoF array type definition
     typedef ModelPart::DofsArrayType DofsArrayType;
 
-    /** Counted pointer of ConvergenceCriteria */
-    KRATOS_CLASS_POINTER_DEFINITION(ConvergenceCriteria);
     ///@}
     ///@name Life Cycle
     ///@{
 
     /** Constructor.
      */
-    ConvergenceCriteria()
+    explicit ConvergenceCriteria()
```

Comment:
make explicit also the copy constructors (for example here on line 94)

Reply:
Ok